### ISSUE TITLE
More SEO features and basic stuff

### ISSUE THREAD

**Issue by axisdos** [xeoncross/forumfive#7]
Hey man, I love your script and its interface. Really modern and clean. However, could you please add a few basic SEO feature such as url friendly, custom title for each post because every page has the same title as the homepage title. 

Also please add subpages and add noindex tag to those subpage too. Thanks

**Comment by xeoncross** [comment]
Already done: https://github.com/Xeoncross/forumfive/blob/master/index.php#L13

It's worth pointing out that the demo is running v1 of the system. The master branch is on v2.

**Comment by axisdos** [comment]
It looks like it was only one of the many features that I mentioned, hopefully you could add more.

**Comment by xeoncross** [comment]
The only other feature I see is "subpages". Could you please explain what you mean since the system has no "pages"?

**Comment by axisdos** [comment]
Here are the features that I suggested the other day:
1. Post title instead of homepage title (Done!)
2. SEO friendly url
3. Noindex for subpages (if you already have subpages (page/1 and page/2 etc)
4. Auto generate meta description for each post (use an excerpt of the post content) instead of homepage description.

**Comment by xeoncross** [comment]
An SEO friendly URL path is an idea I would love to do, but it will require making the system larger and an Apache/Nginx rewrite setup.

I still want to know what you mean by a "subpage" - and why you would want to ban search engines from a subpage.

**Comment by axisdos** [comment]
Allowing search engines to index subpages would cause duplicate content and it will greatly affect the site ranking. I hope you could add more SEO features, thanks.

I don't know if you know WordPress, in their Admin CP option there is a feature where you can show how many posts could be display per page. And so it you set 5 posts and your site have 10 posts in total, that would create a subpage (domain.com/page/2).

**Comment by xeoncross** [comment]
Oh, you mean "pagination". Yes, pagination is something important not only for SEO but also server load. That is on my todo list once I figure out a lightweight way to handle it. 

However, additional result pages would not have duplicate content issues because they would all be additional comments with different text than the initial page.

**Comment by axisdos** [comment]
Yes, but it wouldn't be helpful if they display on search engines because they're subpages. :)

**Comment by xeoncross** [comment]
The only reason they would display in search results is because more people linked to the subpage (apparently it was better content) or the subpage is the only page that mentions the keywords the person is looking for.

I don't see any reason to block them and lose visitors.

**Comment by axisdos** [comment]
You could take a look at http://webmasters.stackexchange.com/questions/51589/any-good-reason-to-noindex-page-2-in-archives and https://yoast.com/articles/wordpress-seo/#noindexfollow

Most people prefer to have noindex for their subpages.

**Comment by xeoncross** [comment]
So, there is a difference between blog post comments and forum pagination.

Blog post comments almost always include the complete blog post (causing duplicate content) while forum pagination does not.

**Comment by axisdos** [comment]
Yes i know, but lets say the forum subpages have the title like: Forum name page 2. How would that be beneficial to search engines? and where is the important keyword(s) in ranking? They're just subpages. Search engines care about the homepage and its posts.

**Comment by xeoncross** [comment]
I don't think you understand how search engine placement works. Page meta tags like title, keywords, and description plan no role in search result placement. They are simply for the benefit of the users reading the result.

You are going to remove an entire page of relative discussion (full of real keywords) from search engines just because they have a title similar to the first topic title...?

**Comment by axisdos** [comment]
I understand your point of view. However, I still wish you could add an option in your script to disallow search engines from indexing subpages.
